src/Makefile

```diff
@@ -731,12 +731,23 @@ install:
 	-cp $(EXE) $(BINDIR)
 	-strip $(BINDIR)/$(EXE)
 
-#clean all
+# clean all
 clean: objclean profileclean
 	@rm -f .depend *~ core
 
+# clean binaries and objects
+objclean:
```

Comment:
these seem to be moved from private targets to public targets.. is that needed?

Reply:
They were already in section 4 of the Makefile (public targets), I just reordered section 4 to put clean, objclean and profileclean targets adjacent again in section 4.

Now the `net` target if after the cleaning targets, do you prefer `net` higher in the list of section 4 ?

Reply:
if we move things probably move objclean and profileclean down to Section 5? However, that's unrelated to the patch, so better not to touch it IMO, it makes reviewing it more difficult.

Reply:
Fixed, thanks